docs/cli-existing-project.md

```diff
@@ -200,8 +199,7 @@ Update your `foundry.toml` file to include a few settings required by Sphinx. We
 ```
 build_info = true
 extra_output = ['storageLayout']
-fs_permissions = [{ access = "read-write", path = "./"}]
-allow_paths = ["../.."]
+fs_permissions=[{access="read", path="./out"}, {access="read-write", path="./cache"}]
```

Comment:
The user may not have an artifact directory named `out` and a cache directory named `cache`. I think we should use placeholder values here, and say something like "Make sure to replace ... with ..."

Reply:
Good point I think I'm just going to revert this change for now since I think this guide is already somewhat involved.